src/libcfitsio.jl

```diff
@@ -929,15 +929,15 @@ for (a,b,T) in ((:fits_get_num_cols,  "ffgncl",  :Cint),
     end
 end
 
-function fits_get_colnum(f::FITSFile, tmplt::String)
+function fits_get_colnum(f::FITSFile, tmplt::String; case_sensitive::Bool=true)
     result = Ref{Cint}(0)
     status = Ref{Cint}(0)
 
     # Second argument is case-sensitivity of search: 0 = case-insensitive
     #                                                1 = case-sensitive
     ccall(("ffgcno", libcfitsio), Cint,
           (Ptr{Void}, Cint, Ptr{UInt8}, Ref{Cint}, Ref{Cint}),
-          f.ptr, 0, tmplt, result, status)
+          f.ptr, Int(case_sensitive), tmplt, result, status)
```

Comment:
`Cint` for clarity, or maybe just leave off the conversion and ccall will convert it automatically.

Reply:
Good point, thanks.